mepo.d/cmdline/parser.py

```diff
@@ -231,6 +231,7 @@ def __commit(self):
         commit = self.subparsers.add_parser(
             'commit',
             description = 'Commit staged files in the specified components')
+        commit.add_argument('-a', action = 'store_true', help = 'stage all tracked files and then commit')
```

Comment:
Could you add `--all` as a long option here? Only because I see that `man git-commit` says that's also true for `git commit`